docs/messaging/kafka.md

```diff
@@ -112,6 +112,15 @@ This section defines how to apply semantic conventions when collecting Kafka met
 | messaging.kafka.leader.elections             | Counter       | Int64      | elections | `{election}` | Leader election rate (increasing values indicates broker failures). | | |
 | messaging.kafka.leader.unclean-elections     | Counter       | Int64      | elections | `{election}` | Unclean leader election rate (increasing values indicates broker failures). | | |
 | messaging.kafka.brokers                      | UpDownCounter | Int64      | brokers   | `{broker}`   | Number of brokers in the cluster. | | |
```

Comment:
This metric now should be removed/deprecated

Reply:
Done, I moved to a new section called `### Broker Metrics` and I removed the old metric as well.